spec/requests/copy_cards_order_completed_forms_spec.rb

```diff
@@ -7,11 +7,6 @@
   describe "GET new_copy_cards_order_completed_form_path" do
     let(:path) { new_copy_cards_order_completed_form_path(transient_registration.token) }
 
-    before do
```

Comment:
Not sure how this is related to logging refatoring?

Reply:
When adding the FeatureToggle for log filtering I had to add lots of stubs for unrelated tests because of the way other FeatureToggle'd features were set up. So rather than add to the existing mess I simplified it.